application/src/test/java/uk/gov/hmcts/net/ccd/definition/store/rest/MultipleControllersEndpointIT.java

```diff
@@ -111,4 +117,44 @@ public void shouldReturnSingleWizardPageForTestComplexAddressBookCase() throws E
                 hasItem(Matchers.hasProperty("caseFieldId", containsString("ContectEmail"))))))
         );
     }
+
+    // To be @Nested - UserRoleController
+    @Test
+    public void shouldReturnUserRolesForDefinedRoles() throws Exception {
+        givenUserProfileReturnsSuccess();
+        final String URL = String.format(ROLES_URL, "CaseWorker1,CaseWorker2,CaseWorker3,Fatih,Andreij, Mario");
+        MvcResult result = mockMvc.perform(MockMvcRequestBuilders.get(URL))
+            .andExpect(MockMvcResultMatchers.status().isOk())
+            .andReturn();
+        final List<UserRole> userRoles = mapper.readValue(result.getResponse().getContentAsString(), TypeFactory.defaultInstance().constructType(new TypeReference<List<UserRole>>(){}));
+        assertThat(userRoles, hasSize(6));
+    }
+
+    // To be @Nested - CaseDefinition Controller
+    @Test
+    public void shouldReturnJurisdictions() throws Exception {
+        givenUserProfileReturnsSuccess();
```

Comment:
Same as above.

Reply:
This is required for the definition import phase